tests/unit/controlgroup/controlgroup_core.js

```diff
@@ -0,0 +1,77 @@
+module( "Controlgroup: Core" );
+
+function hasCornerClass( element ) {
```

Comment:
Yup but probably `has/lacksClassStart` instead. This test was the inspiration for those assertions. They didn't exist at the time this was written though.